public/stock-controller.js

```diff
@@ -1,12 +1,12 @@
 (function() {
     'use strict';
 
-    angular.module('openfin.stock', ['openfin.quandl'])
-       .controller('StockCtrl', ['$routeParams', 'quandlService', function($routeParams, quandlService) {
+    angular.module('openfin.stock', ['openfin.quandl', 'openfin.openfin'])
+       .controller('StockCtrl', ['$routeParams', 'quandlService', 'openfinService', function($routeParams, quandl, openfin) {
```

Comment:
Smart people say that you shouldn't rename your controllers. Source: https://github.com/mgechev/angularjs-style-guide . This is in a few more places.

Quote:

>If using array definition syntax, use the original names of the controller's dependencies. This will help you produce more readable code: